backend/src/main/java/capstone/myfinancemanager/manager/controller/UserController.java

```diff
@@ -0,0 +1,31 @@
+package capstone.myfinancemanager.manager.controller;
+
+import capstone.myfinancemanager.manager.model.User;
+import capstone.myfinancemanager.manager.service.UserServiceImp;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@RequestMapping("/auth/register")
+public class UserController {
+    private final UserServiceImp userServiceImp;
+
+    public UserController(UserServiceImp userServiceImp) {
+        this.userServiceImp = userServiceImp;
+    }
+
+    @PostMapping
+    public ResponseEntity<User> registerNewUser(@RequestBody User newUser) {
```

Comment:
Ich würde hier eine andere Klasse benutzen. Mit man aus dem Frontend auf keinen Falls dem timestamp etc setzten kann.
```suggestion
    public ResponseEntity<User> registerNewUser(@RequestBody NewUser newUser) {
```